airgun/views/contentview.py

```diff
@@ -192,6 +256,58 @@ def is_displayed(self):
         )
 
 
+class ContentViewVersionDetailsView(BaseLoggedInView):
+    breadcrumb = BreadCrumb()
+
+    @property
+    def is_displayed(self):
+        breadcrumb_loaded = self.browser.wait_for_element(
+            self.breadcrumb, exception=False)
+        return (
+                breadcrumb_loaded
+                and len(self.breadcrumb.locations) > 3
+                and self.breadcrumb.locations[0] == 'Content Views'
+                and self.breadcrumb.locations[2] == 'Versions'
+        )
+
+    @View.nested
+    class yum_repositories(SatSecondaryTab, SearchableViewMixin):
+        TAB_NAME = 'Yum Repositories'
+        table = SatTable('.//table')
+
+    @View.nested
+    class rpm_packages(SatSecondaryTab):
+        TAB_NAME = 'rpm Packages'
+        repo_filter = SatSelect(".//select[@ng-model='repository']")
+        searchbox = Search()
+        table = SatTable(".//table")
+
+        def search(self, query, repo=None):
```

Comment:
seeing that type of search not for first time. Is it time consuming procedure to make it generic?

Reply:
Submitted https://github.com/SatelliteQE/airgun/issues/205 . Imo it's not that big deal, but requires update in many places and lots of retesting.